src/weather.js

```diff
@@ -0,0 +1,36 @@
+import weatherColor from './colors';
+import {
+  loader, overlay, mainInput, weatherInfo,
+} from './dom';
+import { openPopup, closePopup } from './events';
+
+async function getWeather(city, units) {
+  const url = `https://cors-anywhere.herokuapp.com/api.openweathermap.org/data/2.5/weather?q=${city}&units=${units}&appid=1a7d76daf845e6d9571a1071f31062f1`;
+  return fetch(url).then(data => data.json());
```

Comment:
If you're using async, then you should use `await`. Any particular reason you chose `async` + `.then`? Requirements state that either `promise` or `async/await` should be used [`REQUIRED`], as specified [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/promises_requirement.png).